src/routes/safe/store/actions/createTransaction.ts

```diff
@@ -79,6 +82,18 @@ export const storeTx = async (tx, safeAddress, dispatch, state) => {
   }
 }
 
+interface CreateTransaction extends WithSnackbarProps {
```

Comment:
do we really expect all snackbar props in createTransaction action?

Reply:
well... despite your point is valid, the current implementation doesn't seem to handle the case where `snackbar` props are not present.

The only place where it's given an empty callback is for the `upgradeSafe` action.

Reply:
the thing is that we don't need to pass the these functions at all, snackbar supports redux: https://github.com/gnosis/safe-react/blob/development/src/logic/wallets/store/actions/fetchProvider.ts